### PR TITLE
feat: Modal, Navigation, Landing page

### PR DESCRIPTION
**Pull request by saurabhchirde**

### CODE REVIEW

**Comment by ashishkatiyar2404** [comment]
Your UI looks awesome Saurabh. Well structured, neat, and clean😎

File: Documentation/docpage-css/navigation.css [hunk 1]
```diff
@@ -1,23 +1,60 @@
...
+.nav-header-mobile{
+    box-shadow: 0px 3px 8px var(--btn-hover);
+}
+
+.desktop-navigation-bar {
+    box-shadow: 0px 3px 8px var(--modal-shadow);
+    min-width: 32.5rem;
+}
+
+.nav-header-desktop>a>img,
```
Comment (ashishkatiyar2404) [inline Documentation/docpage-css/navigation.css]: give space like 
```suggestion
.nav-header-desktop > a > img,
```

File: components/card/card.css [hunk 2]
```diff
@@ -198,6 +198,14 @@
     row-gap: 1rem;
 }
 
-.card-shadow {
+.card-shadow-one {
     box-shadow: 0 10px 15px var(--text-light);
+}
+
+.card-shadow-two {
```
Comment (ashishkatiyar2404) [inline components/card/card.css:205]: I have seen you already use color variable try to use  here also

**Review by ashishkatiyar2404 — APPROVED** [review]
Suggest some changes.
your code looks good to go.💯

**Review by Cl0udCode — COMMENTED** [review]
clean!

File: Documentation/docpage-css/navigation.css [hunk 3]
```diff
@@ -1,23 +1,60 @@
...
+}
+
+.nav-header-mobile>div>a>.logo-icon{
+    width: 1.6rem;
+}
+
+.desktop-navigation-bar>[class*="-search-input"] {
+    flex-grow: 1;
+    margin: 0 1rem 0 2rem;
+}
```
Comment (rahulyadav139) [inline Documentation/docpage-css/navigation.css]: You used CSS selectors extensively which is great if you don't want to create irrelevant CSS classes. UI is super clean. Files are clean and readable. Great work. Go ahead and merge the branch

**Review by rahulyadav139 — APPROVED** [review]

**Review by abhijeetekad — APPROVED** [review]
Looking awesome . Great work..

**Review by harshkotkar5557 — APPROVED** [review]
Looks good to me

File: components/card/card.css (hunk 2 above)
Comment (saurabhchirde) [reply components/card/card.css:205]: Updated and created variables for the shadow that has only one color.

**Pull request merged**